lessons/equivalence-and-laws/langs/en-us/index.adoc

```diff
@@ -0,0 +1,167 @@
+|===
+
+Or like this:
+
+[.embedded, cols="^.^3,^.^1,^.^3", grid="none", stripes="none" frame="none"]
+|===
+|@show{(coe '(+ 5 (+ 6 m)))}	| @math{=} | @show{(coe '(+ (+ 5 6) m))}
+| @math{5 + (2 + m)} 			| @math{=} | @math{(5 + 2) + m}
+|===
+
+The Associative Property will hold no matter what values we substitute in for @math{c} or for @math{m}.
+
+The same goes for the Commutative Property! We can represent it with variables, like this:
+
+[.embedded, cols="^.^3,^.^1,^.^3", grid="none", stripes="none" frame="none"]
+|===
+|@show{(coe '(* 5 g))}	| @math{=} | @show{(coe '(* g 5))}
+| @math{5 * g} 			| @math{=} | @math{g * 5}
+|===
+
+Or like this:
+
+[.embedded, cols="^.^3,^.^1,^.^3", grid="none", stripes="none" frame="none"]
+|===
+|@show{(coe '(+ 12 h))}	| @math{=} | @show{(coe '(+ h 12))}
+| @math{12 + h} 			| @math{=} | @math{h + 12}
+|===
+
+It doesn't matter what values we use for @math{g} or for @math{h} - the equations above are _always_ true.
+
+These examples above show us that the Commutative and Associative properties are in fact more powerful than sheer computation. @math{12 + h} and @math{h + 12} can't be proven equivalent by computing the answer – because without knowing h we can't find the answer! Commutativity lets us prove them as equivalent even with variables.
+
+=== Investigate
+
+All of the practice below invites students to apply their knowledge of the Associative and Commutative Properties in equations and Circles that include variables. For some students, this cognitive leap can be a challenging one. The familiarity of the worksheets (they all parallel those in the previous section) can make this transition to more abstract thinking a bit smoother.
+
+Students are also welcome and encouraged to choose values to represent the variables if they in fact want to evaluate the expressions as they are working. Early finishers can substitute in numbers of their choosing to confirm that their analyses of the Circles are correct.
+
+[.lesson-instruction]
+- Complete TRUE or FALSE with Variables using your knowledge of the Associative Property and the Commutative Property.
+- Complete Label the Arrow with Variables. Identify if each Circle Transformation represents the Commutative Property, the Associative Property, or Neither.
+- Analyze the Circles to decide Which One Doesn’t Belong? with Variables. Be sure to explain your thinking.
+
+// ADD A PEDAGOGY BOX FOR "WHICH ONE DOESN'T BELONG" ; there's some nice stuff here: //
+// https://illustrativemathematics.blog/2018/04/10/what-is-an-instructional-routine/ //
+// http://www.meaningfulmathmoments.com/which-one-doesnt-belong-wodb.html //
+
+
+
+
+=== Synthesize
+
+- Are equations with variables always true?
+** _No. If we're looking at an equation that represents one of our Circle Transformations, then yes. An equation such as @math{j - y = y - j} is only sometimes true. And an equation like @math{g \times 0 = 600} is never true._
```

Comment:
I think you're still learning too hard on "Circle" as a vocabulary word. I get that you're trying to show how cool Circles are, but I think this actually works against that goal. The circles are just a way of highlighting structure, and we want teachers to see them as a small cost with a big benefit. Inventing vocabulary like "Circle Transformations" makes it seem like a larger cost: should teachers quiz kids on what Circles Transformations are? And since Circles aren't in the standards, this feels like a bigger cost with zero benefit.

We're looking at equations that differ according to properties of Commutativity and Associativity. When they are validly applied to get from one equation to another, we know they are equivalent. When they are _not_ applied validly, we don't know if they are equivalent.  You can make the same point without adding vocabulary